server/controllers/users/login.js

```diff
@@ -0,0 +1,36 @@
+/* eslint-disable no-console */
```

Comment:
Never disable no-console for an entire file, if it was absolutely needed, then disable it for that single line console.